test/extensions/access_loggers/wasm/config_test.cc

```diff
@@ -56,8 +56,8 @@ TEST_P(WasmAccessLogConfigTest, CreateWasmFromEmpty) {
 }
 
 TEST_P(WasmAccessLogConfigTest, CreateWasmFromWASM) {
-#if defined(__aarch64__)
-  // TODO(PiotrSikora): There are no Emscripten releases for arm64.
+#if !defined(__x86_64__)
```

Comment:
Can we define our own macros in bazel rules (like `ENVOY_WASM_ENABLE_CPP_TESTS`) for these? So when we change how these are enabled we don't need to change all of the tests.

Reply:
I did something similar (#19641), but it's quite a big refactor of tests and the core change here is completely lost in the  refactoring, so I'd prefer if we could merge this as-is for the fixed `#define`, and then get the refactored tests merged.

Reply:
ok that sounds good to me